_tuning-your-cluster/seperate-index-and-search-workloads.md

```diff
@@ -171,3 +173,23 @@ POST my_index/_scale
 }
 ```
 {% include copy-curl.html %}
+
+#### Search replica recovery scenarios
+
+OpenSearch handles recovery of search replicas in search-only mode differently depending on your configuration:
+
+**Scenario 1: Persistent data directory with remote store state disabled**
+
+When you have persistent data and `cluster.remote_store.state.enabled` set to `false`, search replicas recover automatically after node restarts.
+
+**Scenario 2: Remote store state enabled without persistent data directory**
+
+When `cluster.remote_store.state.enabled` is `true` but you don't have a persistent data directory, search replicas recover without primaries and write replicas. Because remote store state is enabled, OpenSearch remembers that the index exists after restart. The allocation logic skips checking for an active primary for search replicas, so search replicas are allocated and search queries remain functional.
+
+**Scenario 3: Remote store state enabled with persistent data directory**
+
+This configuration provides seamless recovery. In search-only mode with a persistent data directory and `cluster.remote_store.state.enabled` set to `true`, OpenSearch correctly brings up only search replicas without primary and regular replicas.
+
+**Scenario 4: No persistent data directory and remote store state disabled**
+
+Without a persistent data directory and with `cluster.remote_store.state.enabled` set to `false`, restarting OpenSearch loses all local state and the index becomes unrecoverable because OpenSearch has no metadata reference and local state is wiped.
```

Comment:
This is only true if you lose all nodes, not just the SR nodes.

Reply:
That is correct, this is true for all replicas irrespective of search replicas or in search_only mode. Just covered this scenario here as I want to mention there is no difference with the behavior.